### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by khizirsiddiqui**
Updates `README.md` as per review in PR#1811

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,37 +119,34 @@ available---for instance, at the time of this writing, Ubuntu 16.04 only has
...
 equivalent to running `./configure` with autotools. If you run CMake with no
-options, it will configure the project to build with no debugging symbols and no
-profiling information:
+options, it will configure the project to build with no debugging symbols and not profiling information: 
 
     $ cmake ../
 
-You can specify options to compile with debugging information and profiling
-information:
+Options can be specified to compile with debugging information and profiling information:
```
[the remark this answers is not in the document]
Comment (khizirsiddiqui) [reply README.md]: There isn't any specific difference except now it is consistent with line 156: `Options are specified with the -D flag.  The allowed options include:`. But yes, they mean exactly the same.

[the remark this answers is not in the document]
Comment (rcurtin) [reply README.md]: That's fair.

File: README.md [hunk 2]
```diff
@@ -187,30 +186,30 @@ Once CMake is configured, building the library is as simple as typing 'make'.
 This will build all library components as well as 'mlpack_test'.
 
     $ make
-
-You can specify individual components which you want to build, if you do not
-want to build everything in the library:
+
```
Comment (rcurtin) [inline README.md]:
```suggestion

```

File: README.md [hunk 3]
```diff
@@ -120,14 +120,14 @@ available---for instance, at the time of this writing, Ubuntu 16.04 only has
...
+There are some useful pages to consult in addition to this section:
 
   - [Building mlpack From Source](https://www.mlpack.org/doc/mlpack-git/doxygen/build.html)
   - [Building mlpack From Source on Windows](https://www.mlpack.org/doc/mlpack-git/doxygen/build_windows.html)
 
 mlpack uses CMake as a build system and allows several flexible build
-configuration options. One can consult any of numerous CMake tutorials for
-further documentation, but this tutorial should be enough to get mlpack built
+configuration options. You can consult any of the CMake tutorials for
+further documentation, but this tutorial should be eough to get mlpack built
```
Comment (zoq) [inline README.md]: This should be "enough" and not "eough".

Comment (rcurtin) [reply README.md]: Nice catch, I missed it :)

**Pull request merged**